docs/src/how-to/feature-selection.rst

```diff
@@ -1,4 +1,21 @@
 Feature Selection
```

Comment:
This should be renamed (this file, the example file and titles) to refer to `properties`, since that's the name equistore uses.